lib/sentry/envelope.ex

```diff
@@ -92,4 +103,15 @@ defmodule Sentry.Envelope do
         throw(error)
     end
   end
+
+  defp item_to_binary(json_library, %ClientReport{} = client_report) do
+    case client_report |> Map.from_struct() |> json_library.encode() do
+      {:ok, encoded_client_report} ->
+        header = ~s({"type": "client_report", "length": #{byte_size(encoded_client_report)}})
```

Comment:
This is silly and I’m noticing I didn't do it for `check_in`s, but let's make the JSON ever so slightly smaller 🙃 

```suggestion
        header = ~s({"type":"client_report","length":#{byte_size(encoded_client_report)}})
```